algorithm.go

```diff
@@ -4,13 +4,13 @@ import "fmt"
 
 const (
 	// HmacSha1 describes a HMAC with SHA-1
-	HmacSha1 Algorithm = 0x01
+	HmacSha1 Algorithm = A_HMAC_SHA1
 
 	// HmacSha256 describes a HMAC with SHA-2 (256-bit)
-	HmacSha256 Algorithm = 0x02
+	HmacSha256 Algorithm = A_HMAC_SHA256
 
 	// HmacSha512 describes a HMAC with SHA-2 (512-bit)
-	HmacSha512 Algorithm = 0x03
```

Comment:
Not sure about this - this is already a public constant. What good does the additional constant do?